services/azure-devops/azure-devops-build.service.js

```diff
@@ -58,22 +65,60 @@ module.exports = class AzureDevOpsBuild extends BaseSvgScrapingService {
         pattern: ':organization/:projectId/:definitionId/:branch',
         namedParams: {
           organization: 'totodem',
-          projectId: '8cf3ec0e-d0c2-4fcd-8206-ad204f254a96',
+          projectId: 'Shields.io',
           definitionId: '2',
           branch: 'master',
         },
         staticPreview: renderBuildStatusBadge({ status: 'succeeded' }),
         keywords,
         documentation,
       },
+      {
+        title: 'Azure DevOps builds (stage)',
+        pattern:
+          ':organization/:projectId/:definitionId/:branch*\\?stage=:stage',
+        namedParams: {
+          organization: 'totodem',
+          projectId: 'Shields.io',
+          definitionId: '5',
+          branch: 'master',
+          stage: 'Successful Stage',
```

Comment:
You can include it inside `queryParams` instead, and simplify the pattern. See [this GitHub badge](https://github.com/badges/shields/blob/0f332413dfc4bfd4feeca7088749c4949f9e980f/services/github/github-release.service.js#L67) for example.

Reply:
That seems indeed way better (I didn't know about it!).